vsp-q3a.php

```diff
@@ -444,7 +444,12 @@ function convertColorCodes($str)
   {
     // parece eliminar efectos del nombre
     if (!strcmp($this->config["gametype"], "xp")) {
-      return $this->convertXPColorCodes($str);
+      $newStr = $this->convertXPColorCodes($str);
```

Comment:
would you mind to use a speaking variable name instead of newStr?
eg convertedStr or something which fits even better?

Reply:
will do